### ISSUE TITLE
Learning rate configuration of shiny blender dataset - Ref NeRF

### ISSUE THREAD

**Issue by Dharmendra04** [google-research/multinerf#151]
I didn't find any configuration on the Shiny Blender dataset in your configuration files on multinerf. Could you please kindly provide me the learning rate configuration you used for the Shiny Blender dataset

What are the initial and final learning rates? Have you used any gradient clipping methods? If so, what are the threshold values you used?

<img width="1106" alt="Screenshot 2024-02-21 at 10 55 23" src="https://github.com/google-research/multinerf/assets/108211571/fae83ef6-ff8c-4112-b419-153d38d6b396">
